ebi_eva_internal_pyutils/mongodb/__init__.py

```diff
@@ -1,3 +0,0 @@
-from ebi_eva_internal_pyutils.mongodb.mongo_database import MongoDatabase
```

Comment:
Is there a particular reason to move this from `ebi_eva_internal_pyutils.mongodb` to `ebi_eva_internal_pyutils`? It's not a big deal but does mean we have to remember to change imports like [this one](https://github.com/EBIvariation/eva-submission/blob/a7a0e876f9568d11f1e0d7446870a0a6da91dcf1/eva_submission/eload_utils.py#L17).

Reply:
I think I moved it by mistake. I've reinstated it.